main.go

```diff
@@ -101,17 +101,19 @@ func main() {
 
 	initializeLogger()
 
-	*filters = strings.Replace(*filters, " ", "", -1)
-	pairs := strings.Split(*filters, ";")
-	for _, pair := range pairs {
-		keyValue := strings.Split(pair, ":")
-
-		// Check format.
-		if len(keyValue) != 2 {
-			panic(fmt.Sprintf("filter '%v' should be of the form `label_key: label_value`", keyValue))
-		}
+	if filters != "" {
```

Comment:
`filters` is a pointer to a string provided by `kingpin`, it needs to be dereferenced
```
	if *filters != "" {
```